roles/create_global_cluster_with_rds_cluster/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Run 'create_global_cluster_with_rds_cluster' role
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+
+  block: 
+    # Error: unable to locate credentials
+    - name: Create global DB cluster
```

Comment:
To fix could not resolve module/action 'amazon.cloud.rds_global_cluster', just add amazon.cloud: '>=0.4.0' here https://github.com/redhat-cop/cloud.aws_ops/blob/main/galaxy.yml#L21